src/main/scala/is/hail/methods/LogisticRegression.scala

```diff
@@ -14,17 +14,16 @@ object LogisticRegression {
     require(vds.wasSplit)
 
     def tests = Map("wald" -> WaldTest, "lrt" -> LikelihoodRatioTest, "score" -> ScoreTest, "firth" -> FirthTest)
```

Comment:
This is a `val` and it shouldn't be inside the `apply` method, it should be a `private` `val` on the `LogisticRegression` object.